graders/python/python_autograder/pl_helpers.py

```diff
@@ -63,24 +63,24 @@ def save_plot(plt: ModuleType, iternum: int = 0) -> None:
             f.write(imgsrc)
 
 
-def points(points: float) -> Callable:
+def points(points: float) -> Callable[..., Any]:
```

Comment:
need to use a typevar here.

Reply:
Fixed with https://github.com/PrairieLearn/PrairieLearn/pull/11251/commits/d0bae0eb5db8d61130d0db600d58ea00e885af97.